### PR TITLE
Update .NET install scripts reference

### LINKED ISSUE #32347: Update --channel current --> sts
Current channel was renamed STS.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: a2e484a9-5e0c-eb3e-1413-7b507c4fa07d
* Version Independent ID: 5bc7caa5-f1c2-d7d5-fe02-80c65cca9672
* Content: [dotnet-install scripts - .NET CLI](https://learn.microsoft.com/en-us/dotnet/core/tools/dotnet-install-script)
* Content Source: [docs/core/tools/dotnet-install-script.md](https://github.com/dotnet/docs/blob/main/docs/core/tools/dotnet-install-script.md)
* Product: **dotnet-fundamentals**
* Technology: **dotnet-cli**
* GitHub Login: @tdykstra
* Microsoft Alias: **tdykstra**

### PR DESCRIPTION
**Pull request by tdykstra**
Fixes #32347

### CODE REVIEW

**Review by IEvangelist — APPROVED** [review]
Looks good to me, we'll :shipit: - thank you 🙏

File: docs/core/tools/dotnet-install-script.md [hunk 1]
```diff
@@ -100,8 +100,8 @@ The install scripts do not update the registry on Windows. They just download th
 
   Specifies the source channel for the installation. The possible values are:
 
-  - `Current` - Most current release.
-  - `LTS` - Long-Term Support channel (most current supported release).
+  - `STS` - Standard Term Support SDK
```
Comment (jamshedd) [inline docs/core/tools/dotnet-install-script.md:103]: There should be no "SDK" after Standard Term Support

File: docs/core/tools/dotnet-install-script.md [hunk 2]
```diff
@@ -100,8 +100,8 @@ The install scripts do not update the registry on Windows. They just download th
...
+  - `LTS` - The most current Long Term Support release
```
Comment (jamshedd) [inline docs/core/tools/dotnet-install-script.md:104]: lets replace the word "current" with recent.

File: docs/core/tools/dotnet-install-script.md (hunk 1 above)
Comment (tdykstra) [reply docs/core/tools/dotnet-install-script.md:103]: I'll change it, but note that this was taken verbatim from the scripts themselves.

File: docs/core/tools/dotnet-install-script.md (hunk 2 above)
Comment (tdykstra) [reply docs/core/tools/dotnet-install-script.md:104]: This change too should be made in the scripts themselves as well.

**Comment by tdykstra** [comment]
@YuliiaKovalova Note changes to --channel help text.

**Comment by YuliiaKovalova** [comment]
Hi @tdykstra ,

The PR with changes is ready. 
https://github.com/dotnet/install-scripts/pull/330
I think we can deploy it tomorrow.

**Pull request merged**